lms/djangoapps/courseware/date_summary.py

```diff
@@ -400,9 +398,7 @@ def register_alerts(self, request, course):
                         '</div>'
                     ).format(
                         upgrade_url=self.link,
-                        upgrade_label=Text(_('Upgrade ({upgrade_price})')).format(
-                            upgrade_price='${upgrade_price}'.format(upgrade_price=upgrade_price),
-                        ),
+                        upgrade_label=Text(_('Upgrade ({upgrade_price})')).format(upgrade_price=upgrade_price),
```

Comment:
Nit: cosmetic_verified_display_price returns a translated string so don't need the _( here